### PR TITLE
Add hello martin file

### PR DESCRIPTION
**Pull request by mkristien**

### CODE REVIEW

File: hello_martin.c [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
+
```
Comment (mkristien) [inline hello_martin.c:2]: good code here

File: hello_martin.c [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+int main() {
+    printf("Hello ajtyvit\n")
```
Comment (mkristien) [inline hello_martin.c:4]:
```suggestion
    printf("Hello ajtyvit\n");
```

**Review by mkristien — COMMENTED** [review]
good code so far

**Review by annaprytysk — APPROVED** [review]
Thanks!

**Pull request merged**